sass/custom/nhs-kent-medway.scss

```diff
@@ -0,0 +1,63 @@
+$govuk-font-family: -apple-system,BlinkMacSystemFont,helvetica neue,helvetica,ubuntu,roboto,noto,segoe ui,arial,sans-serif; // Override font as so not to use New Transport
+@import "application";
+
+// CUSTOM BRANDING
+// Use the variables below to control the style
+// Make sure banner colours meet minimum colour contrast levels
+
+$custom-banner-colour: #ffffff;
+$custom-banner-border-colour: #ffffff;
+$custom-text-colour: #ffffff;
```

Comment:
This probably wants to be black or no-one will be able to see it.

Reply:
Thanks, Alex. Amended now.